docs/output.md

```diff
@@ -6,29 +6,50 @@ This document describes the output produced by the pipeline. Most of the plots a
 
 The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
-
 ## Pipeline overview
 
 The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
 
-* [FastQC](#fastqc) - Raw read QC
-* [MultiQC](#multiqc) - Aggregate report describing results and QC from the whole pipeline
-* [Pipeline information](#pipeline-information) - Report metrics generated during the workflow execution
+- [FastQC](#fastqc) - Raw read QC
+- [falco](#fastqc) - Alternative to FastQC for raw read QC
+- [fastp](#fastp) - Adapter trimming for Illumina data
+- [AdapterRemoval](#adapterremoval) - Adapter trimming for Illumina data
+- [Porechop](#porechop) - Adapter removal for Oxford Nanopore data
+- [BBDuk](#bbduk) - Quality trimming and filtering for Illumina data
+- [PRINSEQ++](#prinseq) - Quality trimming and filtering for Illunina data
+- [Filtlong](#filtlong) - Quality trimming and filtering for Nanopore data
+- [Bowtie2](#bowtie2) - Host removal for Illumina reads
+- [minimap2](#minimap2) - Host removal for Nanopore reads
+- [SAMtools stats](#samtools-stats) - Statistics from host removal
+- [SAMtools bam2fq](#samtools-fastq) - Converts unmapped BAM file to fastq format (minimap2 only)
```

Comment:
Is the link for `samtools bam2fq` displayed in VS?

Reply:
oops!